pkg/loghttp/push/otlp_config.go

```diff
@@ -146,8 +146,8 @@ func (c *AttributesConfig) UnmarshalYAML(unmarshal func(interface{}) error) erro
 }
 
 type ResourceAttributesConfig struct {
-	IgnoreDefaults   bool               `yaml:"ignore_defaults,omitempty"`
-	AttributesConfig []AttributesConfig `yaml:"attributes,omitempty"`
+	IgnoreDefaults   bool               `yaml:"ignore_defaults,omitempty" doc:"description=Configure whether to ignore the default list of Resource Attributes to be stored as Index Labels and only use the given Resource Attributes config"`
+	AttributesConfig []AttributesConfig `yaml:"attributes_config,omitempty"`
```

Comment:
I have renamed the config to avoid some confusion since, otherwise, it was causing the `resource_attributes` block to have two sections named `attributes` as below:
```
resource_attributes:
  attributes:  <----
    - action: index_label
      attributes:  <----
        - service.group
```

With this change, it would look like:
```
resource_attributes:
  attributes_config: 
    - action: index_label
      attributes: 
        - service.group
```